### DISCUSSION TITLE
Problems when modeling the radial velocity curve with PHOEBE using different reference times

### DISCUSSION THREAD

**Discussion by mloyarce** [phoebe-project/phoebe2#917] [Questions]
Hello PHOEBE Team,

I have the following doubt/question: I am working on modeling the radial velocity curve and, to do so, I initially provide PHOEBE with the orbital period and the reference time, to phase the curve. The problem is that, depending on the reference time it uses, PHOEBE may or may not be able to model the curve, as can be seen in the attached images. The first two images correspond to a T0 (the first image shows the estimator model and the second the optimizer model), while the last two images correspond to the same thing, but with a different T0. Each T0 is related to one of the eclipses in the light curve. Unfortunately, the eclipse I need PHOEBE to use to center the curve corresponds to T0 at which PHOEBE does not correctly model the radial velocity curve. This is because, in the other eclipse, there is a greater contribution from a disk present in the system. I do not understand why PHOEBE works with one T0 but not the other, since, to the naked eye, the curve looks "the same" (although the radial velocity measurements change their position in phase, the shape of the curve remains the same). 

It is worth mentioning that for this I am only using the RV data set in PHOEBE, as so far, I have not been able to simultaneously model the RV and LC. 

Thank you for your help.
Maria Laura
![Screenshot from 2024-07-01 10-38-48](https://github.com/phoebe-project/phoebe2/assets/171082839/9f6b9143-09e2-4da5-9d57-3f5a3f613e0d)
![Screenshot from 2024-07-01 10-39-02](https://github.com/phoebe-project/phoebe2/assets/171082839/038f1aff-b0e9-46c2-8251-39576176b285)
![Screenshot from 2024-07-01 10-39-58](https://github.com/phoebe-project/phoebe2/assets/171082839/05671bf3-74f8-4e59-9e86-deb0cd0ff675)
![Screenshot from 2024-07-01 10-58-52](https://github.com/phoebe-project/phoebe2/assets/171082839/8e38bcba-750a-4af1-bfcf-f2754b8d1773)

**Comment by Raindogjones** [comment]
Hi Maria Laura,
It is difficult to diagnose without more detail but it looks to me as though the issue might be with the period more than with T0. The first model looks like it is fitting with a period different to the period you are phasing with - otherwise I don't understand how the model for, e.g., the primary RVs can be decreasing between phase -0.3 and -0.2 before rising and then decreasing.  Not sure why this might be, but it looks as though also your definition of T0 is different between the 2 solutions - for the second it is clearly the superior conjunction, for the first it seems to be the inferior conjunction (which would correspond to the eclipse of the secondary). So perhaps the T0 of your first model is off by half a period?
Maybe take a look at the T0 docs to make sure you have everything set up correctly: http://phoebe-project.org/docs/2.4/tutorials/t0s

**Reply by mloyarce** [reply]
Hi David,

Thanks for the prompt response. I'm setting T0 with the twig 't0_supconj' in both cases. Indeed, the fit period is different than the period I'm using to phase the curve in the first model. Why does this happen and why does it not happen with the second model? And I tried with a T0 shifted by half of the period and I got the same results. Could it be that I'm not computing well the T0 for the secondary eclipse (like it can be a completely different value,i.e, I'm using 7731 d and it should be 5670 d)?

**Reply by Raindogjones** [reply]
Ok, so if you are setting your t0 by t0_supconj, your phase data should look like the lower plots with the primary RVs declining through phase zero - this is why I suggested that perhaps your t0 in the upper plots was off by 0.5 in phase.  It could also be that you have your primary and secondary incorrectly identified in the RVs (i.e. the primary RVs are actually those of the secondary, and vice versa).

In any case, once you have the t0_supconj and period from your light curves, you likely don't want to fit them using the RVs. You don't have that many points so they probably won't constrain them too well.  So, you could try fixing those and just fitting for vgamma, sma and q.

**Reply by mloyarce** [reply]
Thank you very much for your help, David! I will be more careful reviewing all I give to PHOEBE.
